src/coreclr/scripts/superpmi_aspnet.py

```diff
@@ -174,7 +174,9 @@ def build_and_run(coreclr_args):
 
         # note tricks to get one element tuples
 
-        runtime_options_list = [("Dummy=0",), ("TieredCompilation=0", ), ("TieredPGO=1", "TC_QuickJitForLoops=1"), ("TieredPGO=1", "TC_QuickJitForLoops=1", "ReadyToRun=0")]
+        runtime_options_list = [("Dummy=0",), ("TieredCompilation=0", ), ("TieredPGO=1", "TC_QuickJitForLoops=1"), ("TieredPGO=1", "TC_QuickJitForLoops=1", "ReadyToRun=0"),
```

Comment:
should first items have `TC_OnStackReplacement=0` as you're planning to enable it by default?

Reply:
I suppose so, though with the current defaults (~50,000 hits to patchpoints before a transition) very few methods will end up with OSR. In practice it won't make much difference.

The bigger impact will be from changing the default for QJFL.